docs/runtime-compliance-testing.md

```diff
@@ -6,4 +6,375 @@ In order to be tested for [compliance][], runtimes MUST support at least one of
 
 * Version 1.0.0 of the [OCI Runtime Command Line Interface](command-line-interface.md).
 
-[compliance]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc4/spec.md#notational-conventions
+## Running the runtime validation suite
+
+The runtime validation suite uses [node-tap][], which is packaged for some distributions (for example, it is in [Debian's `node-tap` package][debian-node-tap]).
+If your distribution does not package node-tap, you can install [npm][] (for example, from [Gentoo's `nodejs` package][gentoo-nodejs]) and use it:
+
+```console
+$ npm install tap
+```
+### From a release
+
+Check if your release has pre-compiled tests on the [release page][releases] page.
+
+```
+$ tar xf runtime-tools-v0.6.0.tar.gz
+$ RUNTIME=runc tap ./runtime-tools-v0.6.0/validation/*.t
+```
+
+### From source
+
+Build the validation executables:
+
+```console
+$ make runtimetest validation-executables
+```
+
+Runtime validation currently [only supports](docs/runtime-compliance-testing.md) the [OCI Runtime Command Line Interface](doc/command-line-interface.md).
```

Comment:
These relative links (and some later ones in this file) need to be updated now that you've moved the text under `docs/`.

Also, this information is generic for runtime-validation; it's not specific to the “From source” approach.  Can you shift it (and some of the text from the next paragraph) up into the “Running the runtime validation suite” section?